packages/wallet-core/db/schemas/1-init.js

```diff
@@ -0,0 +1,37 @@
+/* AUTO GENERATED FILE - DON'T CHANGE THE CONTENT */
+
+export default {
+  "schemaVersion": 1,
+  "schema": [
+    {
+      "name": "test",
+      "primaryKey": "id",
+      "properties": {
+        "id": {
+          "type": "int"
+        },
+        "name": {
+          "type": "string"
+        }
+      }
+    },
+    {
+      "name": "wallet",
+      "primaryKey": "id",
+      "properties": {
+        "id": {
+          "type": "int"
+        },
+        "name": {
+          "type": "string"
+        },
+        "publicKey": {
```

Comment:
publicKey should probably be 'address'

Reply:
Sure, 'address' is better. I will change this name.

I tried to replicate the same model from the desktop wallet, that's why it was created this way.

https://github.com/SelfKeyFoundation/Identity-Wallet/blob/master/src/main/wallet/wallet.js#L24

Reply:
yes, it is wrong there, will be changed when we'll have time for it